### PR TITLE
Lt 1564 1560 1559

### PR DESCRIPTION
**Pull request by phlptt**

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -225,3 +226,19 @@
...
+if DEBUG:
+    # Disable security features in local environment.
+    SECURE_BROWSER_XSS_FILTER = False
+    SESSION_COOKIE_SECURE = False
+    CSRF_COOKIE_SECURE = False
+else:
+    # Enable security features in hosted environments.
+    SECURE_BROWSER_XSS_FILTER = True
+    SESSION_COOKIE_SECURE = True
+    CSRF_COOKIE_SECURE = True
```
Comment (craiga) [inline conf/settings.py]: These could all be `not DEBUG`.

```python
SECURE_BROWSER_XSS_FILTER = not DEBUG
SESSION_COOKIE_SECURE = not DEBUG
CSRF_COOKIE_SECURE = not DEBUG
```

File: conf/settings.py [hunk 2]
```diff
@@ -54,6 +54,7 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'authbroker_client.middleware.ProtectAllViewsMiddleware',
+    'conf.middleware.TestMiddleware',
```
Comment (craiga) [inline conf/settings.py]: What is this?

Comment (a-bek) [reply conf/settings.py]: Forgotten gems

File: conf/settings.py [hunk 3]
```diff
@@ -225,3 +225,19 @@
         },
     }
 }
+
+
+# Security settings
+SECURE_CONTENT_TYPE_NOSNIFF = True
+SECURE_BROWSER_XSS_FILTER = True
+
+if DEBUG:
```
Comment (a-bek) [inline conf/settings.py]: You can replace the whole if / else block with:

```
SECURE_BROWSER_XSS_FILTER = not DEBUG
SESSION_COOKIE_SECURE = not DEBUG
CSRF_COOKIE_SECURE = not DEBUG
```

**Pull request merged**